packages/twenty-server/src/engine/core-modules/file/controllers/file.controller.ts

```diff
@@ -1,4 +1,12 @@
-import { Controller, Get, Param, Req, Res, UseGuards } from '@nestjs/common';
+import {
+  Controller,
+  Get,
+  Param,
+  Req,
+  Res,
+  UseFilters,
+  UseGuards,
+} from '@nestjs/common';
```

Comment:
```suggestion
import {
  Get,
  Req,
  Res,
  Param,
  UseGuards,
  Controller,
  UseFilters,
} from '@nestjs/common';
```

i think arranging in this pattern make it easier to reviewing the code  and helps to increase the code quality its just a suggestion

Reply:
The sorting should be done automatically via Linter rules :) (alphabetically, not by length)